recipes/molgrid/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -ex
+
+# hack for now: https://github.com/conda-forge/pyquaternion-feedstock/pull/1
+pip install https://github.com/KieranWynn/pyquaternion/archive/refs/tags/v0.9.9.tar.gz
```

Comment:
This should be addressed before a merge

Reply:
If you have the permission to merge it I think https://github.com/conda-forge/pyquaternion-feedstock/pull/1 is quick to review. It's just a rebuild since the package does not exist.